sceptre/cli/helpers.py

```diff
@@ -0,0 +1,267 @@
+import contextlib
+import os
+import sys
+from functools import wraps
+from json import JSONEncoder
+
+import click
+import logging
+import yaml
+
+from boto3.exceptions import Boto3Error
+from botocore.exceptions import BotoCoreError, ClientError
+from jinja2.exceptions import TemplateError
+
+from sceptre.exceptions import SceptreException, RecursiveFlagMissingError
+from sceptre.exceptions import StackPathRequiredError, StackConfigNotFoundError
+from sceptre.stack_status_colourer import StackStatusColourer
+from sceptre.environment import Environment
+
+
+def catch_exceptions(func):
+    """
+    Catches and simplifies expected errors thrown by sceptre.
+
+    catch_exceptions should be used as a decorator.
+
+    :param func: The function which may throw exceptions which should be
+        simplified.
+    :type func: func
+    :returns: The decorated function.
+    :rtype: func
+    """
+    @wraps(func)
+    def decorated(*args, **kwargs):
+        """
+        Invokes ``func``, catches expected errors, prints the error message and
+        exits sceptre with a non-zero exit code.
+        """
+        try:
+            return func(*args, **kwargs)
+        except (SceptreException, BotoCoreError, ClientError, Boto3Error,
+                TemplateError) as error:
+            write(error)
+            sys.exit(1)
+
+    return decorated
+
+
+def write(var, output_format="str", no_colour=True):
+    """
+    Writes ``var`` to stdout. If output_format is set to "json" or "yaml",
+    write ``var`` as a JSON or YAML string.
+
+    :param var: The object to print
+    :type var: obj
+    :param output_format: The format to print the output as. Allowed values: \
+    "str", "json", "yaml"
+    :type output_format: str
+    :param no_colour: Whether to color stack statuses
+    :type no_colour: bool
+    """
+    if output_format == "json":
+        encoder = CustomJsonEncoder()
+        stream = encoder.encode(var)
+    if output_format == "yaml":
+        stream = yaml.safe_dump(var, default_flow_style=False)
+    if output_format == "str":
+        stream = var
+
+    if not no_colour:
+        stack_status_colourer = StackStatusColourer()
+        stream = stack_status_colourer.colour(stream)
+
+    click.echo(stream)
+
+
+@contextlib.contextmanager
+def change_set(stack, name):
```

Comment:
It seems to me like there is a better place for this function than here.

Reply:
Can remove all together